src/services/llms/anthropic.py

```diff
@@ -0,0 +1,82 @@
+from typing import List, Dict
+from anthropic import Anthropic
+from unidiff import Hunk, PatchedFile
+from ...core.config import Config
+from ...core.models import PRDetails
+from .base import BaseLLMService
+
+class AnthropicService(BaseLLMService):
+    """
+    Implementation of BaseLLMService for Anthropic's Claude models.
+    """
+    
+    def __init__(self):
+        """Initialize the Anthropic client with configuration."""
```

Comment:
Chỗ này nên thêm xử lý lỗi nếu `Config.ANTHROPIC_API_KEY` không được đặt.  Ví dụ:  `raise ValueError("ANTHROPIC_API_KEY is not set in the configuration.")`